docs/getting_started/configuration/prawini.rst

```diff
@@ -26,7 +26,7 @@ user defined ``praw.ini`` files in a few other locations:
    .. note:: To check the values of the environment variables, you can
       open up a terminal (Terminal/Terminal.app/Command Prompt/Powershell)
       and echo the variables (replacing <varaible> with the name of the 
```

Comment:
while we're at it, "varaible" -> variable

Reply:
Tip: You can make suggestions, so I can directly commit the fix, such as: 
```suggestion
      and echo the variables (replacing <variable> with the name of the 
```